video_xblock/static/html/brightcove.html

```diff
@@ -18,7 +18,7 @@
         data-embed="default"
         data-application-id
         data-setup='{"playbackRates": [0.5, 1, 1.5, 2] }'
-        width="829"
+        width="100%"
         height="511"
```

Comment:
This is confusing. Width=100%, height=511px. Will video have correct proportions?
Maybe we can remove `height` altogether?

Here and in similar places.